src/SourceTerms/Deleptonization/CPU_Src_Deleptonization.cpp

```diff
@@ -207,7 +214,13 @@ static void Src_Deleptonization( real fluid[], const real B[],
       const int  NTarget1 = 3;
 #     endif
             int  In_Int1[NTarget1+1];
-            real In_Flt1[3], Out1[NTarget1+1];
+#     ifdef TEMP_IG
+            real In_Flt1[4];
+                 In_Flt1[3] = Temp_IG_Kelv;
```

Comment:
Same question as that in `CPU_EoS_Nuclear.cpp`.

Also, why put the initialization of `In_Flt1[3]` here?